### PR TITLE
3MFLoader: Clean up

### PR DESCRIPTION
**Pull request by Mugen87**
Minor clean up...

### CODE REVIEW

File: examples/js/loaders/3MFLoader.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/**
+ * @author technohippy / https://github.com/technohippy
+ */
+
 THREE.ThreeMFLoader = function ( manager ) {
```
Comment (mrdoob) [inline examples/js/loaders/3MFLoader.js]: What do you think about renaming it to `TMFLoader`? Just like `TDSLoader` is for 3DS...?

**Comment by mrdoob** [comment]
Thanks!

File: examples/js/loaders/3MFLoader.js (hunk 1 above)
Comment (Mugen87) [inline examples/js/loaders/3MFLoader.js]: Um, not sure this naming convention is intuitive :thinking:. Although `ThreeMFLoader` looks a bit strange, i think it is still a clearer name than `TMFLoader`...

Comment (looeee) [inline examples/js/loaders/3MFLoader.js]: Why is the 3DS loader called TDSLoader in the first place?

Comment (Mugen87) [inline examples/js/loaders/3MFLoader.js]: Good question! :wink:

Comment (mrdoob) [inline examples/js/loaders/3MFLoader.js]: It's a javascript thing...

screen shot 2017-08-11 at 4 14 15 pm

Comment (looeee) [inline examples/js/loaders/3MFLoader.js]: OK, that makes sense. But it's still not very intuitive - I had gone over the loaders quite a few times before I realised there was a 3ds loader. I just assumed that tds was some obscure format I had never heard of.

Comment (mrdoob) [inline examples/js/loaders/3MFLoader.js]: Would it make more sense if we use this pattern for loaders that start with a number?

```js
new THREE['3DSLoader']()
```

**Pull request merged**